src/components/Search/Search.jsx

```diff
@@ -0,0 +1,30 @@
+import { useRef } from 'react';
+
+function Search(props) {
+	const { searchCB } = props;
```

Comment:
Might be simpler to lift the search handler to the parent component to get the value